SQL.test.js

```diff
@@ -104,26 +105,26 @@ test('SQL helper - build complex query with map', async t => {
   }
 
   const values = SQL.map(objArray, mapFunction)
-  t.equal(values !== null, true)
+  assert.equal(values !== null, true)
   const sql = SQL`INSERT INTO users (id) VALUES (${values})`
 
-  t.equal(sql.text, 'INSERT INTO users (id) VALUES ($1,$2,$3)')
-  t.equal(sql.sql, 'INSERT INTO users (id) VALUES (?,?,?)')
-  t.equal(sql.debug, 'INSERT INTO users (id) VALUES (1,2,3)')
-  t.same(sql.values, [1, 2, 3])
+  assert.equal(sql.text, 'INSERT INTO users (id) VALUES ($1,$2,$3)')
+  assert.equal(sql.sql, 'INSERT INTO users (id) VALUES (?,?,?)')
+  assert.equal(sql.debug, 'INSERT INTO users (id) VALUES (1,2,3)')
+  assert.deepStrictEqual(sql.values, [1, 2, 3])
 })
 
 test('SQL helper - build complex query with map - using default mapper function', async t => {
   const ids = [1, 2, 3]
 
   const values = SQL.map(ids)
-  t.equal(values !== null, true)
+  assert.equal(values !== null, true)
   const sql = SQL`INSERT INTO users (id) VALUES (${values})`
 
-  t.equal(sql.text, 'INSERT INTO users (id) VALUES ($1,$2,$3)')
-  t.equal(sql.sql, 'INSERT INTO users (id) VALUES (?,?,?)')
-  t.equal(sql.debug, 'INSERT INTO users (id) VALUES (1,2,3)')
-  t.same(sql.values, [1, 2, 3])
+  assert.equal(sql.text, 'INSERT INTO users (id) VALUES ($1,$2,$3)')
```

Comment:
why not the assertions that come on the test context t?

Reply:
To still support node 18.
Assertion with test context is a node 20 feature.
If that's ok for you to drop also node 18 I can change it